packages/core/src/routes/experience/classes/libraries/sign-in-experience-validator.ts

```diff
@@ -98,7 +102,10 @@ export class SignInExperienceValidator {
       }
       case InteractionEvent.Register: {
         assertThat(
-          signInMode !== SignInMode.SignIn,
+          signInMode !== SignInMode.SignIn ||
+            // This guarantees new users can still be created through one-time token
+            // authentication even if the registration is turned off.
+            (type === VerificationType.OneTimeToken && isVerified),
```

Comment:
See the comments above.  Maybe declaring an exclusive `isOnetimeTokenVerified` parameter is better than allowing any verification record as an input.

Reply:
Updated